test/unit/audience/multipleAudienceTypes.test.ts

```diff
@@ -0,0 +1,134 @@
+import { edkt } from '../../../src';
+import {
+  getMatchedAudiences,
+  getPageViews,
+  clearStore,
+} from '../../helpers/localStorage';
+import {
+  cosineSimAudience,
+  logRegAudience,
```

Comment:
Would be nice to see the parts of the definition used in this test - just a small comment, but I mean if this was a function, we could call it with the values in this file - so it would be easy to see :)